flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/api/internal/TableEnvImpl.scala

```diff
@@ -412,6 +412,18 @@ abstract class TableEnvImpl(
       case dropTable: SqlDropTable =>
         val objectIdentifier = catalogManager.qualifyIdentifier(dropTable.fullTableName(): _*)
         catalogManager.dropTable(objectIdentifier, dropTable.getIfExists)
+      case createFunction: SqlCreateFunction =>
+        val operation = SqlToOperationConverter
+          .convert(planner, createFunction)
+          .asInstanceOf[CreateFunctionOperation]
+        val identifier = catalogManager.qualifyIdentifier(createFunction.fullFunctionName(): _*)
+        catalogManager.createFunction(
+          operation.getCatalogFunction,
+          identifier,
+          operation.isIgnoreIfExists)
+      case dropFunction: SqlDropFunction =>
+        val identifier = catalogManager.qualifyIdentifier(dropFunction.fullFunctionName(): _*)
+        catalogManager.dropFunction(identifier, dropFunction.getIfExists)
```

Comment:
do you need to convert it to `DropFunctionOperation`? do we foresee this to happen in the future that `DropFunctionOperation.getFunctionPath` returns something different from `SqlDropFunction.fullFunctionName`? 

e.g. will this work?
```
CREATE FUNCTION default_catalog.db.func1 AS 'my.org.Func`
DROP FUNCTION func1
```

Reply:
catalogManager.qualifyIdentifier will actually compose a full path with the current catalog and current database in this case.